app/controllers/app.py

```diff
@@ -337,14 +337,18 @@ def post(self, project_id):
 
         app_data = request.get_json()
 
+        app_data['project_id'] = project_id
+
+        print(app_data)
+
         validated_app_data, errors = app_schema.load(app_data)
 
         if errors:
             return dict(status='fail', message=errors), 400
 
         existing_app = App.find_first(
             name=validated_app_data['name'],
-            project_id=project_id)
+            project_id=validated_app_data['project_id'])
```

Comment:
And this?

Reply:
pics from what was added up. I just put it back to its default because that's what was there before.